### ISSUE TITLE
Add dark/light mode set on default on linux

### ISSUE THREAD

**Issue by jeffconrad18** [brave/brave-browser#29308]
Please add dark/light mode set on default on linux. That like on Windows the colour mode is set as dark when the dark mode of ubuntu or another linux distro is activated. I already tried the GTK theme but it doesn't change the "brave colour". It's very useful for the dark reader that changes the light of a side when brave colour is toggled.

**Comment by rebron** [comment]
Hi @jeffconrad18 I wanted to clarify. Did you want a setting that automatically picks up what mode you're using dark mode on Linux then use Dark mode for Brave automatically?

**Comment by jeffconrad18** [comment]
@rebron Yes!

**Comment by Rikiub** [comment]
Yes, please! Brave in Windows has the option to follow the "system theme" and it is very useful. Firefox has this option also following the current [GTK theme](https://wiki.archlinux.org/title/Firefox#Dark_themes).

![image](https://github.com/brave/brave-browser/assets/114118274/3f799213-382a-4981-84df-5ff726144e46)

**Comment by posita** [comment]
Duplicate of #14685 (open for 2.5 years as of this post)?

**Comment by popy2k14** [comment]
> Bump! Now that upstream chromium has support for the system-wide dark theme preference (as per Freedesktop / XDG Desktop Spec), it should be easy to pull from upstream. Chromium 116.0.5845.140 already works, brave 1.57.62 with upstream Chromium 116.0.5845.180 cannot.

+1

**Comment by fallaciousreasoning** [comment]
Hey maybe I'm misunderstanding this issue, but this looks like it already works (at least in Ubuntu):
[Screencast from 2024-07-11 08-50-38.webm](https://github.com/brave/brave-browser/assets/7678024/46c5d67f-c582-4d2a-b356-a8036625bdcb)

**Comment by rebron** [comment]
Closing as stale (fixed) per last comment.
